Dockerfile

```diff
@@ -1,34 +1,33 @@
-FROM node:12-stretch
-LABEL description="Debian image to build nativefier apps"
+FROM node:12-alpine
+LABEL description="Alpine image to build nativefier apps"
 
-# Get wine32, not 64, to work around binary incompatibility with rcedit.
-# https://github.com/nativefier/nativefier/issues/375#issuecomment-304247033
-# Forced us to use Debian rather than Alpine, which doesn't do multiarch.
-RUN dpkg --add-architecture i386
 
 # Install dependencies
-RUN apt-get update \
-    && apt-get --yes install wine32 imagemagick \
-    && apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apk update \
+    && apk add bash wine imagemagick dos2unix \
+    && rm -rf /var/lib/apk/lists/*
```

Comment:
Hmmmmmmmm are you sure this is the path for `apk` cache? (I'm surprised it'd exactly at the same spot, except for t → k)

Reply:
It is not. I thought in an example I was following that's what they had, but I may have gotten led astray. Will fix.